runtime/oti/j9nonbuilder.h

```diff
@@ -4998,6 +4998,7 @@ typedef struct J9VMContinuation {
 	UDATA* j2iFrame;
 	struct J9JITGPRSpillArea jitGPRs;
 	struct J9VMEntryLocalStorage* oldEntryLocalStorage;
+	J9VMThread *carrierThread;
```

Comment:
I know this line is a copy-paste from a discussion, but in this file '*' is kept next to the type (rather then next to the field) name.